src/components/photo_gallery.js

```diff
@@ -0,0 +1,145 @@
+import _ from 'lodash';
+import React, {Component, PropTypes} from 'react';
+import shouldComponentUpdate from 'react-addons-shallow-compare';
+
+import Row from 'muicss/lib/react/row';
+import Col from 'muicss/lib/react/col';
+import GridList from 'material-ui/lib/grid-list/grid-list';
+import RaisedButton from 'material-ui/lib/raised-button';
+import FontIcon from 'material-ui/lib/font-icon';
+
+import GalleryItem from './photo_gallery_item';
+
+export default class PhotoGallery extends Component {
+  static propTypes () {
+    return {
+      photos: PropTypes.array.isRequired,
+      editable: PropTypes.bool.isRequired,
+      onChange: PropTypes.func.isRequired,
+      showDialog: PropTypes.func
+    };
+  }
```

Comment:
Can replace it. Just a habit since I did a lot of composite react classes where the propTypes is a function